plover/config.py

```diff
@@ -358,9 +362,13 @@ def _set(self, section, option, value):
 
     def _lookup(self, key):
         name = key[0] if isinstance(key, tuple) else key
-        opt = self._OPTIONS[name]
-        if opt.full_key is not None:
-            key = opt.full_key(self, key)
+        try:
+            opt = self._OPTIONS[name]
+            if opt.full_key is not None:
+                key = opt.full_key(self, key)
+        except KeyError:
+            log.error('invalid config option: %r', name, exc_info=True)
+            return key, null_option()
```

Comment:
What are those changes (and the one above) for?

Reply:
That code logs an error if the provided config option can't be found. Since the user is now able to attempt to set config values themselves through a command, I figured logging an error and returning a do-nothing option would be better than a KeyError that propagates out and crashes the program. The "null option" is just a dummy option with the required methods that does nothing at all; it allows the surrounding code to attempt to get and set operations without returning errors.